mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -2830,9 +2831,7 @@ def _generate_auto_accept_response(cls, length, page_size, active=None, number_o
                             'active': not j,
                             'reason': f'I am an automatically generated auto-accept number {i}',
                             'created': (
-                                datetime.datetime.now(
-                                    tz=pytz.timezone('Europe/London')
```

Comment:
`'Europe/London'` shouldn't be hard-coded as there's a setting for local timezone already